com/lijo/basics/calculator.java

```diff
@@ -0,0 +1,46 @@
+import java.util.*;
+public class calculator {
```

Comment:
Class Name should always start with a capital letter and also should match the file name
`Calculator.java`

Reply:
okay sir